scalalib/src/mill/javalib/android/AndroidSdkModule.scala

```diff
@@ -118,13 +119,47 @@ trait AndroidSdkModule extends Module {
     PathRef(buildToolsPath().path / "apksigner")
   }
 
+  /**
+   * Provides the path for the Android Debug Bridge (adt) tool.
+   *
+   * For more information, refer to the official Android documentation [[https://developer.android.com/tools/adb]]
+   */
+  def adbPath: T[PathRef] = Task {
+    PathRef(sdkPath().path / "platform-tools" / "adb")
```

Comment:
You can consolidate multiple path segments into one string as long as it's a literal

```scala
PathRef(sdkPath().path / "platform-tools/adb")
```

Here and below